aiida/manage/external/rmq.py

```diff
@@ -154,7 +155,7 @@ def handle_continue_exception(node, exception, message):
 
         if not node.is_excepted and not node.is_sealed:
             node.logger.exception(message)
```

Comment:
Is there a reason thy this goes to the logger rather than to the process report as in the other case?

Could one factor out these two lines into a private node member function that is called in both places?

Reply:
The `report` method is on the `Process` class, not `ProcessNode`. `report` then calls the process logger, which in practice should always be the node logger.

But I agree it would be good to streamline more